frank-flow/src/frontend/src/app/editor/monaco-editor/monaco-editor.component.ts

```diff
@@ -90,15 +90,15 @@ export class MonacoEditorComponent implements AfterViewInit, OnDestroy {
           });
         };
 
-        if (!(window as any).require) {
+        if ((window as any).require) {
+          onAmdLoader();
+        } else {
           const loaderScript: HTMLScriptElement =
             document.createElement('script');
           loaderScript.type = 'text/javascript';
           loaderScript.src = 'assets/monaco/vs/loader.js';
           loaderScript.addEventListener('load', onAmdLoader);
           document.body.append(loaderScript);
-        } else {
-          onAmdLoader();
         }
```

Comment:
Is this really needed?
Pretty sure webpack loads it in a certain way so as long as angular uses webpack to bundle modules it feels a bit redundant to check what loader is used.

Reply:
I wouldn't know.
The linter changed this becouse I touched the file.
I think it's fine to leave this for now. But I hope that we will load the Monaco editor in a different way in the future.

Reply:
Maybe an issue should be made for this

Reply:
Good idea: https://github.com/frankframework/frank-flow/issues/1277